CMakeLists.txt

```diff
@@ -222,6 +222,47 @@ elseif((FREERTOS_PORT STREQUAL "A_CUSTOM_PORT") AND (NOT TARGET freertos_kernel_
     "       freertos_kernel)")
 endif()
 
+########################################################################
+# Requirements
+set(CMAKE_C_STANDARD 99)
```

Comment:
@phelter  Why do you need C99? We want to support C89. Did you notice anything C99 specific in the kernel?

Reply:
Thank you for the clarification. I did build the kernel with `-Wdeclaration-after-statement` and I see no warning from kernel source. I will remove the following 2 lines which were added in this PR - 
```
set(CMAKE_C_STANDARD 99)
set(CMAKE_C_STANDARD_REQUIRED ON)
```

Reply:
Here is the PR - https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/633